projects/angular/src/forms/combobox/_combobox.clarity.scss

```diff
@@ -50,132 +34,59 @@
-      clr-combobox-border-radius,
-      combobox-variables.$clr-combobox-border-radius,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      border-bottom-right-radius,
-      clr-combobox-border-radius,
-      combobox-variables.$clr-combobox-border-radius,
-      variables.$clr-use-custom-properties
-    );
+    border-bottom-left-radius: combobox-variables.$clr-combobox-border-radius;
+    border-bottom-right-radius: combobox-variables.$clr-combobox-border-radius;
   }
   .clr-focus {
     width: 100%;
   }
   .clr-error .clr-focus-indicator {
-    @include mixins.css-var(
-      background-color,
-      clr-forms-invalid-color,
-      forms-variables.$clr-forms-invalid-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: forms-variables.$clr-forms-invalid-color;
   }
 
   .clr-combobox-wrapper {
     position: relative;
 
     //Dimensions
-    padding: 0 caret-gap() 0 combobox-variables.$clr-combobox-trigger-left-padding;
-    @include mixins.css-var(
-      padding-left,
-      clr-combobox-padding-left,
-      combobox-variables.$clr-combobox-trigger-left-padding,
-      variables.$clr-use-custom-properties
-    );
-
-    @include mixins.css-var(
-      min-height,
-      clr-combobox-min-height,
-      combobox-variables.$clr-combobox-min-height,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      min-width,
-      clr-combobox-min-width,
-      combobox-variables.$clr-combobox-min-width,
-      variables.$clr-use-custom-properties
-    );
+    padding: 0 caret-gap() 0 combobox-variables.$clr-combobox-trigger-padding-left;
+    padding-left: combobox-variables.$clr-combobox-padding-left;
+
+    min-height: combobox-variables.$clr-combobox-min-height;
+    min-width: combobox-variables.$clr-combobox-min-width;
     .clr-input {
-      padding-right: 0;
-      @include mixins.css-var(
-        padding-left,
-        clr-combobox-input-left,
-        variables.$clr_baselineRem_0_25,
-        variables.$clr-use-custom-properties
-      );
-      // ATM remove
-      @include mixins.css-var(
-        max-height,
-        clr-combobox-input-height,
-        variables.$clr_baselineRem_1_25,
-        variables.$clr-use-custom-properties
-      );
+      padding: 0;
+      max-height: tokens.$cds-global-space-7;
+
       &:not(:disabled) {
-        @include mixins.css-var(color, clr-combobox-text-color, inherit, variables.$clr-use-custom-properties);
+        color: combobox-variables.$clr-combobox-text-color;
       }
     }
-    @include mixins.css-var(
-      font-size,
-      clr-combobox-font-size,
-      combobox-variables.$clr-combobox-font-size,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(font-weight, clr-combobox-font-weight, inherit, variables.$clr-use-custom-properties);
+    font-size: combobox-variables.$clr-combobox-font-size;
+    font-weight: combobox-variables.$clr-combobox-font-weight;
```

Comment:
For refactoring ticket `font-size` and `font-weight` should be moved to `.clr-input { ... }` rule.

Reply:
Removing them leads to a long line of "find the half pixel regression" game. Already spent too much time on it. And this is code that will most certainly go away during DOM refactor anyway.